.idea/dictionaries/tiqr.xml

```diff
@@ -0,0 +1,18 @@
+<component name="ProjectDictionaryState">
```

Comment:
This file is in .gitignore, but still committed. Is this deliberate?

Reply:
Yes. The folder `.idea` is already ignored, but `.idea/dictionaries` is whitelisted, so this file gets added.

Reply:
Indeed, I misread the rule :)